streampipes-data-explorer-commons/src/main/java/org/apache/streampipes/dataexplorer/commons/sanitizer/MeasureNameSanitizer.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.streampipes.dataexplorer.commons.sanitizer;
+
+import java.util.List;
+
+public class MeasureNameSanitizer {
+
+  private final List<String> forbiddenChars = List.of(
+      "/"
```

Comment:
Didn't we also need to exclude `-`? 🤔 (cc @tenthe)
We also might need to replace other characters like `&`, did you already check that?

Reply:
Hey,

I have tested several special characters. Here are the results:
<img width="990" alt="grafik" src="https://github.com/apache/streampipes/assets/5279561/b74ebe04-2d5c-4dd3-b47e-6d1e0f3a357e">

It seems that `?`, `"`, `=` should also be escaped.
Cheers, 
Philipp

Reply:
Thanks @tenthe for providing the tests!